pkg/envoy/api/http_connection_manager.go

```diff
@@ -35,7 +35,7 @@ import (
 
 // NewHTTPConnectionManager creates a new HttpConnectionManager that points to the given
 // RouteConfig for further configuration.
-func NewHTTPConnectionManager(routeConfigName string, enableAccessLog, enableProxyProtocol bool) *hcm.HttpConnectionManager {
+func NewHTTPConnectionManager(routeConfigName string, enableAccessLog, enableProxyProtocol bool, idleTimeout time.Duration) *hcm.HttpConnectionManager {
```

Comment:
Could you change the arguments to take `config.Kourier` instead of `enableAccessLog, enableProxyProtocol bool, idleTimeout time.Duration`?